vllm/config.py

```diff
@@ -143,6 +145,13 @@ def __init__(
         self._verify_quantization()
         self._verify_cuda_graph()
 
+        self.extra_inputs_spec: ExtraModelInputsSpec = getattr(
```

Comment:
The model's config can specify if the model requires any additional inputs. This is then used in ModelRunner to capture CUDA graphs with extra inputs and execute the model.